appscale/tools/admin_api/version.py

```diff
@@ -184,6 +200,22 @@ def from_xml(root):
         except StandardError:
             raise AppEngineConfigException('Invalid app.yaml: automatic_scaling invalid.')
 
+    # Adds optional elements for automatic scaling.
+    try:
```

Comment:
Shouldn't this be in the automatic-scaling section?

Reply:
Hmm .. I thought I put it there: the previous code is for automaticScaling.standardSchedulerSettings and if I understand correctly the idle options are in automaticScaling only.

Reply:
The `elif` block applies to the entire `automatic_scaling` section. (`automatic_scaling = root.find(qname('automatic-scaling'))`). In addition, you'll want to update the automatic_scaling dict if it exists rather than overwrite it since minInstances may have been populated.

Reply:
ah! Thanks!